plugins/doc_fragments/redis.py

```diff
@@ -49,6 +49,20 @@ class ModuleDocFragment(object):
       - Path to root certificates file. If not set and O(tls) is
         set to V(true), certifi ca-certificates will be used.
     type: str
+    aliases:
+      - tls_ca_cert_file
```

Comment:
Please remove the aliases. The option names are close enough to the Redis command line options so that users can associate them. (Generally aliases should only be added in very special situations.)